### PR TITLE
Update Generator for v10

### PR DESCRIPTION
**Pull request by JRJurman**
## Summary

PR to update the scaffolding for Tram-One v10. This project setup is simpler, while also having more examples of the API (and links to the docs and discord). 

## Checklist
- [x] PR Summary
- [x] Tests
- [x] Version Bump

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,21 +19,25 @@ $ npx tram-one-express app-name
...
-│   │       ├── ColorHeader.test.js
-│   │       └── index.js
+│   ├── app-header.js
+│   ├── app-summary.js
+│   ├── app-task-description.js
+│   ├── app-task-item.js
+│   ├── app-task-list.js
 │   ├── index.js
-│   └── styles.css
-├── .gitignore
```
Comment (chtinahow) [inline README.md:34]: Why was this removed?

Comment (JRJurman) [reply README.md:34]: mistake in how `tree` command was used, it actually is laid down, good catch!

File: template/src/app-header.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+/**
+ * Simple component that uses children in the rendered tag.
+ * Read more about it here: https://tram-one.io/#components
+ */
+
+const html = registerHtml()
+
+export default (props, children) => {
+	return html`
+    <h1 class="app-header">
```
Comment (chtinahow) [inline template/src/app-header.js:12]: Is it just the view here, or is the indentation weird here?

File: template/src/app-task-list.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+export default () => {
+	const taskList = useTaskList()
+
+	const taskListItems = taskList.map(
+		task => html`
+			<app-list-item task-id=${task.id} />
+		`
+	)
+	return html`
+    <div class="app-task-list">
```
Comment (chtinahow) [inline template/src/app-task-list.js:23]: indentation looks weird here too

File: template/src/index.js [hunk 4]
```diff
@@ -1,22 +1,38 @@
...
-import useColor from './hooks/useColor'
+
+/**
+ * The entry point of the app, and where we mount the app on the DOM.
+ * Read more about it here: https://tram-one.io/#components
+ */
 
 const html = registerHtml({
-	ColorHeader
+	'app-header': AppHeader,
```
Comment (chtinahow) [inline template/src/index.js:16]: indentation?

**Review by chtinahow — APPROVED** [review]
Just the indentation comment, but otherwise LGTM 👍

File: template/src/index.js (hunk 4 above)
Comment (JRJurman) [reply template/src/index.js:16]: previously the project inconsistently has tabs and spaces. It should now all be tabs (this will be better enforced with a linting tool).

File: template/src/app-header.js (hunk 2 above)
Comment (JRJurman) [reply template/src/app-header.js]: previously the project inconsistently has tabs and spaces. It should now all be tabs (this will be better enforced with a linting tool).

File: template/src/app-task-list.js (hunk 3 above)
Comment (JRJurman) [reply template/src/app-task-list.js]: previously the project inconsistently has tabs and spaces. It should now all be tabs (this will be better enforced with a linting tool).
